sdk/storage/azure-storage-blob-cryptography/src/test/java/com/azure/storage/blob/specialized/cryptography/EncryptedBlockBlobAsyncApiTests.java

```diff
@@ -939,7 +939,22 @@ public void downloadACFail(OffsetDateTime modified, OffsetDateTime unmodified, S
     }
 
     @Test
-    @Disabled
+    public void downloadStream() {
+        StepVerifier.create(bec.upload(DATA.getDefaultBinaryData())
+                .then(FluxUtil.collectBytesInByteBufferStream(bec.downloadStream())))
+            .assertNext(r -> assertArraysEqual(DATA.getDefaultBytes(), r))
+            .verifyComplete();
+    }
+
+    @Test
+    public void downloadContent() {
```

Comment:
Did we also want a test case for `downloadContentWithResponse()`? I noticed that was used in the previous commit but does not have a test case in this update.